src/js/components/Form/stories/FieldWithComponentProp.js

```diff
@@ -54,23 +65,40 @@ export const FieldWithComponentProp = () => (
         />
         <FormField
           label="Size"
+          aria-label="size"
```

Comment:
Can we still add `htmlFor` and `id`? Although `aria-label` is 'doing the work', ideally down the road we want Select to work with `htmlFor` and `id` so it would be good to demonstate having these in the story.
```suggestion
          label="Size"
          htmlFor="size"
          id="size"
          aria-label="size"
```